### PR TITLE
Change SimpleListItem to text, added custom styles to download button

### PR DESCRIPTION
**Pull request by matoval**
Change SimpleListItem to text because SimpleListItem is returning an error that a is not a function. SimpleListItem still functions correctly but sends an error to the console. I also added custom styles to the download link so it matches the update button.

### CODE REVIEW

**Comment by resoluteCoder** [comment]
looking good :+1: and its running tests!! :1st_place_medal:

**Comment by boaz0** [comment]
@matoval what is the error message? Also do we have a ticket open for it?

Thanks.

**Comment by matoval** [comment]
> @matoval what is the error message? Also do we have a ticket open for it?
> 
> Thanks.

The error message is "a is not a function" and it's coming from SimpleListItem.js.

I'll make a ticket. When we make tickets for bugs what Epic do we link it to?

**Comment by boaz0** [comment]
What browser are you using?
I am using Firefox and don't see it?

**Comment by matoval** [comment]
> What browser are you using?
> I am using Firefox and don't see it?

I'm using Chrome, the error is in the console when I click the download button.

File: src/App.scss [hunk 1]
```diff
@@ -49,3 +49,11 @@
 .add-100vh {
   height: 100vh
 }
+
+.force-text-black {
+  color: var(--pf-c-dropdown__menu-item--Color) !important;
```
Comment (boaz0) [inline src/App.scss:54]: It seems like `!important` is not necessary here; without it I still see the link black.
And while you are at it, can you delete these lines:
```css
.force-padding-left {
  padding-left: 1rem !important;
}
```

no component is using this abomination :vomiting_face:  anymore

**Review by boaz0 — CHANGES_REQUESTED** [review]
Thank god we can get rid of `!important` from our code base :raised_hands: :tada:

**Review by boaz0 — APPROVED** [review]
LGTM :+1:

**Pull request merged**